src/Symfony/Bridge/Twig/Tests/AppVariableTest.php

```diff
@@ -186,11 +186,12 @@ public function testGetFlashes()
         $flashMessages = $this->setFlashMessages();
         $this->assertEquals($flashMessages, $this->appVariable->getFlashes(array()));
 
-        $flashMessages = $this->setFlashMessages();
```

Comment:
I don't understand why this line was removed. It's important to always initialize the flash messages. Imagine that our code is wrong and returns the first type of flash messages when the type doesn't exist. Before we detected that problem and now we can't.